tests/tickets/4527/1.js

```diff
@@ -7,7 +7,6 @@
 	bender.editor = {
 		editorName: 'editor1',
 		startupData: '',
-		name: document.title,
```

Comment:
Why is this change introduced?

Reply:
This one leads to errors in IE. This name was used inside `querySelector`. Unfortunately in IE, there were some restricted characters in the selector. The same goes for https://github.com/ckeditor/ckeditor4/pull/4601/files#diff-a15eeadc2752af4c0b9dcfc057fb5fe450dfb6cf9a672e4fc9baacaac9036903

So both tests start to fail because of this.